cursive-core/src/printer.rs

```diff
@@ -1,3 +1,4 @@
+#![warn(missing_docs)]
```

Comment:
I don't think we need this, since it's already set at the crate level.

Reply:
Ok, then I will remove it. :)